src/routes/root.jsx

```diff
@@ -0,0 +1,23 @@
+import { App } from "../App.jsx"
+import {Outlet, Link} from "react-router-dom";
+
+export default function Root() {
+     return (
+         <>
+            <li>
+                <Link to={'/subpage'}>
```

Comment:
To avoid writing out route paths in mutliple places we could create a separate file for the router that also exports these paths